File: ffmpeg/src/main/java/com/coder/ffmpeg/utils/Direction.java
package com.coder.ffmpeg.utils;

import androidx.annotation.IntDef;

/**
 * @author: AnJoiner
 * @datetime: 19-12-19
 */
@IntDef({
        Direction.LAYOUT_HORIZONTAL,
        Direction.LAYOUT_VERTICAL,
})
public @interface Direction {
    int LAYOUT_HORIZONTAL = 1;
    int LAYOUT_VERTICAL = 2;
}

File: ffmpeg-mini/src/main/java/com/coder/ffmpeg/annotation/Direction.kt
package com.coder.ffmpeg.annotation

import androidx.annotation.IntDef

/**
 * @author: AnJoiner
 * @datetime: 19-12-19
 */
@IntDef(Direction.LAYOUT_HORIZONTAL,
        Direction.LAYOUT_VERTICAL)
annotation class Direction {
    companion object {
        const val LAYOUT_HORIZONTAL = 1
        const val LAYOUT_VERTICAL = 2
    }
}